recipe/no_async.patch

```diff
@@ -0,0 +1,82 @@
+diff -Naur pexpect-4.1.0.orig/pexpect/async.py pexpect-4.1.0/pexpect/async.py
```

Comment:
This is an unfortunate way of doing it because you'll need to regenerate the patch any time the file changes, but there we go.